src/components/appointments/mocks.js

```diff
@@ -7,9 +8,37 @@ const DEFAULT_PRODUCTS = [
   {code: 'RIJAAN', identifier: 'e8e045ab', name: 'Rijbewijs aanvraag (Drivers license)'},
 ];
 
-const DEFAULT_LOCATIONS = [{identifier: '1', name: 'Maykin Media'}];
+const LOCATIONS = [
+  {
+    products: ['166a5c79', 'e8e045ab'],
+    location: {identifier: '1396f17c', name: 'Open Gem'},
+  },
+  {
+    products: ['e8e045ab'],
+    location: {identifier: '34000e85', name: 'Bahamas'},
+  },
+];
+
+const _getDate = (numDaysToAdd = 0) => {
+  return () => {
+    const now = new Date();
+    const newDate = numDaysToAdd ? addDays(now, numDaysToAdd) : now;
+    return formatISO(newDate, {representation: 'date'});
+  };
+};
```

Comment:
this is a closure so that the `new Date()` is at run time and not at module-load time, to be able to use `jest.useFakeTimer({now: ...})` in tests, which gives stable dates to make assertions about, while still providing dynamic dates in storybook so that next year you don't need to click the "left" icon 10 times to see the available dates.